### PR TITLE
SSH support

### PR DESCRIPTION
**Pull request by aughr**
Set :ssh to true to connect via SSH, :ssh_user with which user. This can be set on a per-environment basis, same as any other setting.

### CODE REVIEW

File: lib/centurion/deploy_dsl.rb [hunk 1]
```diff
@@ -180,13 +180,20 @@ def tls_paths_available?
...
+  def build_server_params
+    opts = {}
+    if fetch(:tlsverify)
+      opts[:tls] = fetch(:tlsverify || tls_paths_available?)
+      opts[:tlscacert] = fetch(:tlscacert)
+      opts[:tlscert] = fetch(:tlscert)
+      opts[:tlskey] = fetch(:tlskey)
+    end
+
+    if fetch(:ssh)
```
Comment (intjonathan) [inline lib/centurion/deploy_dsl.rb]: Running `set(:ssh, 'false')` would probably evaluate true in this line. Maybe `if fetch(:ssh, false) == true`?

File: lib/centurion/docker_server.rb [hunk 2]
```diff
@@ -18,15 +18,15 @@ class Centurion::DockerServer
                  :remove_container, :restart_container
   def_delegators :docker_via_cli, :pull, :tail, :attach, :exec, :exec_it
 
-  def initialize(host, docker_path, tls_params = {})
+  def initialize(host, docker_path, params = {})
```
Comment (intjonathan) [inline lib/centurion/docker_server.rb]: Could this be `cxn_params` or `connection_opts`? `params` is up there with `data` for bad varnames.

File: lib/centurion/docker_via_api.rb [hunk 3]
```diff
@@ -2,13 +2,20 @@
...
 require 'securerandom'
+require 'centurion/ssh'
 
 module Centurion; end
 
 class Centurion::DockerViaApi
-  def initialize(hostname, port, tls_args = {}, api_version = nil)
-    @tls_args = default_tls_args(tls_args[:tls]).merge(tls_args.reject { |k, v| v.nil? }) # Required by tls_enable?
-    @base_uri = "http#{'s' if tls_enable?}://#{hostname}:#{port}"
+  def initialize(hostname, port, params = {}, api_version = nil)
```
Comment (intjonathan) [inline lib/centurion/docker_via_api.rb]: There's `params` again.

**Comment by aughr** [comment]
@intjonathan 👍  Thanks for the feedback, all addressed.

**Comment by aughr** [comment]
@intjonathan Done!

**Comment by aughr** [comment]
Bumped version to 1.9.0 as well.

**Pull request merged**